client/app/host/QuestionAccordion.tsx

```diff
@@ -0,0 +1,96 @@
+'use client';
+import { Switch } from '@/components/ui/switch';
+import {
+    Accordion,
+    AccordionContent,
+    AccordionItem,
+    AccordionTrigger,
+} from '@/components/ui/accordion';
+
+interface QuestionAccordionProps {
+    questions: {
+        question: string;
+        description: string;
+        language: string;
```

Comment:
This should probably be 
```suggestion
        language?: string[];
```